Formula/no-more-secrets.rb

```diff
@@ -0,0 +1,16 @@
+class NoMoreSecrets < Formula
+  desc 'Recreates the SETEC ASTRONOMY effect from "Sneakers"'
```

Comment:
But it is rendered in plain text in a terminal, so I can't italicize in this case. (Well, I *could* in some terminals, but we don't have markup for that, and I'm loath to put raw ANSI terminal-control codes in a description.) I think *some* typography is called for to indicate that it's a title, and not just a proper noun like a restaurant name. I chose double-quotes as a second-best substitute for italics; this seems to be a common convention when more advanced typesetting isn't available, at least in American English. And I've usually seen it in double quotes, not single quotes.

Happy to switch to escaped double quotes inside double quotes — it started out like this but I flipped it to single quotes on the outside just because it looked ugly and I thought folks would actually prefer the single-quote/double-quote inversion to backslashes. In the Ruby books I've read, that use is touted as one of the motivations for the interchangeability of double and single quotes in Ruby.

One more item for the Prose Style Guide, I guess. ;)